opentelemetry-jaeger-propagator/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## vNext
+
+## v0.1.0
+
+### Added
+
+- Move opentelemetry-jaeger::Propagator to new crate: opentelemetry-jaeger-propagator [#1487](https://github.com/open-telemetry/opentelemetry-rust/pull/1487)
```

Comment:
Wording suggestion: Please add something to the tune of
_This is a new create containing just the propagator functionality that was originally part of opentelemetry-jaeger._